prepare_zephyr.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -xeu
+
+ZEPHYR_SHA="42fccba"
+ZEPHYR_SOURCE_URL="https://github.com/zephyrproject-rtos/zephyr/archive/${ZEPHYR_SHA}.zip"
+
+wget -Ozephyr.zip "$ZEPHYR_SOURCE_URL"
+unzip zephyr.zip
+rm zephyr.zip
+mv zephyr-* zephyr
+
+west init -l zephyr
+west update
```

Comment:
I would prefer having a separate directory for `west`, `modules`, `tools` and `zephyr`. Usually we just go with `zephyr`, and have paths with `zephyr/zephyr` inside, but that's ok. 

This would reduce the gitignore and would keep the working dir more clean

Reply:
also the sdk could go inside as well

Reply:
Now there is `zephyr` directory in which both source (`zephyr` dir) and west/sdk (`sdk` dir) are populated.